<ui version="4.0" >
 <class>TrackEditor2Class</class>
 <widget class="QMainWindow" name="TrackEditor2Class" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>24</y>
     <width>1030</width>
     <height>830</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QTreeView" name="treeView" >
        <property name="minimumSize" >
         <size>
          <width>324</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>508</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea" >
        <property name="widgetResizable" >
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>498</width>
           <height>600</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="nemaText" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="action_Read_Log" />
    <addaction name="action_Load_Track" />
    <addaction name="action_Save_Track" />
    <addaction name="action_Save_Track_As" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>854</y>
     <width>1030</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Read_Log" >
   <property name="text" >
    <string>&amp;Read Log</string>
   </property>
  </action>
  <action name="action_Save_Track" >
   <property name="text" >
    <string>&amp;Save Track</string>
   </property>
  </action>
  <action name="action_Save_Track_As" >
   <property name="text" >
    <string>Save Track &amp;As</string>
   </property>
  </action>
  <action name="action_Load_Track" >
   <property name="text" >
    <string>&amp;Load Track</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
